x-pack/plugins/dataset_quality/public/components/flyout/dataset_summary.tsx

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { FieldFormatsStart } from '@kbn/field-formats-plugin/public';
+import { ES_FIELD_TYPES, KBN_FIELD_TYPES } from '@kbn/field-types';
+import {
+  flyoutDatasetCreatedOnText,
+  flyoutDatasetDetailsText,
+  flyoutDatasetLastActivityText,
+} from '../../../common/translations';
+import { DataStreamStat } from '../../../common/data_streams_stats/data_stream_stat';
+import { DataStreamDetails } from '../../../common/data_streams_stats/data_stream_details';
+import { FieldsList } from './fields_list';
+
+interface DatasetSummaryProps {
+  fieldFormats: FieldFormatsStart;
+  dataStreamDetails: DataStreamDetails;
+  dataStreamStat: DataStreamStat;
+}
+
+export function DatasetSummary({
+  dataStreamStat,
+  dataStreamDetails,
+  fieldFormats,
+}: DatasetSummaryProps) {
+  const dataFormatter = fieldFormats.getDefaultInstance(KBN_FIELD_TYPES.DATE, [
+    ES_FIELD_TYPES.DATE,
+  ]);
+  const formattedLastActivity = dataFormatter.convert(dataStreamStat.lastActivity);
+  const formattedCreatedOn = dataStreamDetails.createdOn
+    ? dataFormatter.convert(dataStreamDetails.createdOn)
+    : '';
```

Comment:
@isaclfreire if we don't have the `Created on` info should we just remove the row in the table?

Reply:
yes, an index without a backing index will not return a `Created on` date. This will most likely be true for data streams without data